charts/athens-proxy/templates/deployment.yaml

```diff
@@ -91,8 +91,10 @@ spec:
         {{- end }}
         ports:
         - containerPort: 3000
-        {{- if eq .Values.storage.type "disk" }}
+        {{- if or (eq .Values.storage.type "disk") .Values.upstreamProxy.enabled .Values.netrc.enabled }}
```

Comment:
Just a question - why is `.Values.upstreamProxy.enabled` included?

Reply:
I believe that's because if the upstream proxy is enabled, the filter file configmap gets mounted into `/usr/local/lib` on line 102.